framework/util/to_string.h

```diff
@@ -284,7 +289,7 @@ inline std::string EnumArrayToString(size_t              count,
         tabCount,
         tabSize,
         [&]() { return pObjs != nullptr; },
-        [&](size_t i) { return '"' + ToString(pObjs[i], toStringFlags, tabCount + 1, tabSize) + '"'; });
+        [&](size_t i) { return '"' + ToString(pObjs[i]) + '"'; });
```

Comment:
Ughh this line means the commit can't be merged without the enums one to come which I have locally stashed.